src/settings.js

```diff
@@ -26,3 +26,7 @@ export function getClients (cozy) {
 export function deleteClientById (cozy, id) {
   return cozyFetchJSON(cozy, 'DELETE', `/settings/clients/${id}`)
 }
+
+export function updateLastSync (cozy) {
+  return cozyFetchJSON(cozy, 'POST', '/settings/synchronized')
```

Comment:
What are data sent in the `POST` request?

---

Quelles sont les données envoyées dans la requête `POST` ?

Reply:
Ohhhh, maybe it is the route to _timestamp_ something. More like a ping or a heartbeat, or something else than _updating last sync_.

Reply:
Well, it updates the last sync date. So maybe `updateLastSyncDate` or `updateLastSyncTimestamp`?